src/libraries/Microsoft.PowerFx.Interpreter/Functions/ArrayUntypedObject.cs

```diff
@@ -61,5 +61,10 @@ public bool TryGetPropertyNames(out IEnumerable<string> result)
             result = null;
             return false;
         }
+
+        public bool TrySetProperty(string name, IUntypedObject value)
+        {
+            throw new NotImplementedException();
+        }

```

Comment:
All of these "Stub" impls are why it may be better to add as a virtual on the class, rather than the interface :)